text/0000-formatted-bliss.md

```diff
@@ -0,0 +1,226 @@
+- Feature Name: `formatted_bliss`
+- Start Date: 2019-05-05
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+1. Introduce a formatting convention (hence: *"the style*")
+   to `rust-lang/rust` (hence: *"the repository*") in a `rustfmt.toml` file.
+
+2. Enforce the style with `src/tools/tidy` in the repository's continuous integration (CI).
+
+3. Add a command `fmt` to rustbuild (`./x.py`).
+   The command formats the repository according to the style.
+
+4. Add a command `@rustbot fmt` on GitHub which will add a commit to a
+   pull request (PR) such that the applied to PR will pass CI in terms of the style.
```

Comment:
I would personally prefer for this to instead rebase and amend each commit, running rustfmt on the individual commits. I'm not sure if that'll potentially cause conflicts (that we would need to manually resolve); if so then I guess the additional commit is our best bet.

Reply:
Can we add a modifier `@rustbot fmt fixup` which does that upon request?

Reply:
Force-pushing the branch also means if the user wants to update their local copy they can't do `git pull` (or equivalents) anymore, which will be confusing for contributors with less git experience.